suite-native/module-authorize-device/src/components/passphrase/PassphraseForm.tsx

```diff
@@ -73,8 +81,29 @@ export const PassphraseForm = ({ inputLabel, onFocus }: PassphraseFormProps) =>
         handleSubmit,
         formState: { isDirty },
         reset,
+        watch,
     } = form;
 
+    const watchPassphrase = watch('passphrase');
+
+    const handleScrollToButton = useCallback(() => {
+        if (scrollView && formWrapperView.current) {
+            // Scroll so the whole form including submit button is visible.The delay is needed, because the scroll can not start before the button animation finishes.
+            formWrapperView.current.measureLayout(scrollView.getInnerViewNode(), (_, y) => {
+                setTimeout(
+                    () => scrollView.scrollTo({ y: y - SCROLL_OFFSET, animated: true }),
+                    SCROLL_DELAY,
+                );
+            });
+        }
+    }, [scrollView]);
+
+    useEffect(() => {
+        if (watchPassphrase !== '') {
```

Comment:
`isDirty` should work be used here to match the render condition for button in JSX.

Reply:
38ecda525e